op-verifier/contracts/OPVerifier.sol

```diff
@@ -1,43 +1,177 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.17;
 
-import { IEVMVerifier } from "@ensdomains/evm-verifier/contracts/IEVMVerifier.sol";
-import { RLPReader } from "@eth-optimism/contracts-bedrock/src/libraries/rlp/RLPReader.sol";
-import { StateProof, EVMProofHelper } from "@ensdomains/evm-verifier/contracts/EVMProofHelper.sol";
-import { Types } from "@eth-optimism/contracts-bedrock/src/libraries/Types.sol";
-import { Hashing } from "@eth-optimism/contracts-bedrock/src/libraries/Hashing.sol";
+import {IEVMVerifier} from '@ensdomains/evm-verifier/contracts/IEVMVerifier.sol';
+import {RLPReader} from '@eth-optimism/contracts-bedrock/src/libraries/rlp/RLPReader.sol';
+import {StateProof, EVMProofHelper} from '@ensdomains/evm-verifier/contracts/EVMProofHelper.sol';
+import {Types} from '@eth-optimism/contracts-bedrock/src/libraries/Types.sol';
+import {Hashing} from '@eth-optimism/contracts-bedrock/src/libraries/Hashing.sol';
+import './interfaces/IDisputeGame.sol';
+import './interfaces/IDisputeGameFactory.sol';
```

Comment:
Same for this

Reply:
Some trick has been implemented to be able to import `@eth-optimism/contracts-bedrock` into the contracts rather than copying the code.